hack/make-devenv.sh

```diff
@@ -19,7 +19,7 @@ run_builder() {
     if ! (docker inspect melange-inception:latest >/dev/null 2>&1 ); then
         set -e
         mkdir _output > /dev/null 2>&1 || : 
-        docker run --rm -v $(pwd):/melange -w /melange \
+        docker run --rm -v "$(pwd):/melange" -w /melange \
```

Comment:
This is superfluous, but if you're going to do it, it would be:

```suggestion
        docker run --rm -v "$(pwd)":/melange -w /melange \
```

Reply:
I would never use spaces in paths but I've known people (especially on Mac) to do so.. in terms of where the quotes should go, both will result in the same expansion but to me, my form is a bit more readable but it's not something I can claim to be important so I've no objection to changing it if you prefer.

Reply:
No.

`"$(pwd):/melange"` and `"$(pwd)":/melange` are two very different constructs from shell grammar perspective.  Your version may be equivalently safe against spaces in practice, but is represented as a single token in the AST, rather than two concatenated tokens.

By isolating the variable expansion into a subtoken, we have more predictable behavior when evaluating the script across various shells.  See also [POSIX XCU section 2.3, item 4](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_02_03), which specifically identifies this quoting behavior.